pkg/tfbridge/validate_input_types.go

```diff
@@ -157,9 +157,8 @@ func (v *PulumiInputValidator) validatePropertyValue(
 	}
 
 	switch typeSpec.Type {
-	case "bool":
-		// The bridge permits coalescing strings to booleans, hence skip strings.
-		if !propertyValue.IsBool() && !propertyValue.IsString() {
+	case "boolean":
```

Comment:
This was "boolean" not "bool" apparently.

Reply:
Can we add an error to `validate.go` to prevent `bool`? The bridge should prevent this kind of user error.

Reply:
Er, not sure what you're suggesting. Pulumi Package Schema mandates "boolean" but we weren't using enums in this code so made a typo. The "bool" branch will never be hit in practice. Or do you mean something else?

Reply:
In the interest of time, I'll merge but address this as a follow up if needed.